src/Schema/Table.php

```diff
@@ -682,17 +602,15 @@ public function getColumn($columnName)
      */
     public function getPrimaryKey()
     {
-        if ($this->_primaryKeyName !== false) {
-            return $this->getIndex($this->_primaryKeyName);
-        }
-
-        return null;
+        return $this->_primaryKeyName !== null
+            ? $this->getIndex($this->_primaryKeyName)
```

Comment:
Please keep it the old way for more accurate code coverage.

Reply:
Code coverage works properly if you separate ternary lines (which I did). Also, ternaries are faster than if-guard clauses.